code/modules/power/lighting.dm

```diff
@@ -35,15 +53,9 @@
 		return
 
 	switch(src.stage)
-		if(1)
-			user << "It's an empty frame."
-			return
-		if(2)
-			user << "It's wired."
-			return
-		if(3)
-			user << "The casing is closed."
-			return
+		if(1) user << "It's an empty frame."

```

Comment:
Change this into a to_chat() please.
Additionally, please put the contents of an if/else/for check under the check itself, for readability.